gluoncv/model_zoo/mixnet.py

```diff
@@ -0,0 +1,515 @@
+        expand_channels = in_channels * expand_ratio
+        se = (se_ratio != 0)
+        self.residual_connection = (stride == 1 and in_channels == out_channels)
+
+        self.body = nn.HybridSequential(prefix='')
+        if expand:
+            self.body.add(_conv1x1(in_channels, expand_channels))
+            self.body.add(norm_layer(in_channels=expand_channels, **(self.norm_kwargs)))
+            self.body.add(Activation(act_type))
+
+        self.body.add(MDConv(expand_channels, kernel_size, stride))
+        self.body.add(norm_layer(in_channels=expand_channels, **(self.norm_kwargs)))
+        self.body.add(Activation(act_type))
+
+        if se:
+            self.body.add(_SE(expand_channels, se_ratio))
+
+        self.body.add(_conv1x1(expand_channels, out_channels))
+        self.body.add(norm_layer(in_channels=out_channels, **(self.norm_kwargs)))
+
+    def hybrid_forward(self, F, x):
+        if self.residual_connection:
+            return x + self.body(x)
+        else:
+            return self.body(x)
+
+
+class MixNet(HybridBlock):
+    r"""MixNet model from the
+    `"MixConv: Mixed Depthwise Convolutional Kernels"
+    <https://arxiv.org/abs/1907.09595>`_ paper.
+
+    Parameters
+    ----------
+    net_type : string, default mixnet_s
+        The name of mixnet models, mixnet_s, mixnet_m and mixnet_l are supported.
+    input_size : int, default 224
+        The size of input image.
+    input_channels : int, default 3
+        Number of channels of input image.
+    stem_channels : int, default 16
+        Number of output channels of the first convolution layer,
+        also means the number of filters of the first convolution layer.
+    feature_size: int, default 1536
+        Number of final channels of the last convolution layer before a classifier.
+    num_classes: int, default 1000
+        Number of classification classes.
+    depth_multiplier: float, default 1.0
+        Update block input and output filters based on depth multiplier.
+    norm_layer : object
+        Normalization layer used (default: :class:`mxnet.gluon.nn.BatchNorm`)
+        Can be :class:`mxnet.gluon.nn.BatchNorm` or :class:`mxnet.gluon.contrib.nn.SyncBatchNorm`.
+    norm_kwargs : dict
+        Additional `norm_layer` arguments, for example `num_devices=4`
+        for :class:`mxnet.gluon.contrib.nn.SyncBatchNorm`.
+    use_global_stats: bool, default False
+        Whether use global moving statistics instead of local batch-norm.
+        This will force change batch-norm into a scale shift operator.
+    """
+    def __init__(self, net_type='mixnet_s', input_size=224, input_channels=3, \
+                 stem_channels=16, feature_size=1536, num_classes=1000, \
+                 depth_multiplier=1.0, norm_layer=BatchNorm, \
+                 norm_kwargs=None, use_global_stats=False, **kwargs):
+        super(MixNet, self).__init__(**kwargs)
+
+    # [in_channels, out_channels, kernel_size, stride, expand_ratio, act_type, se_ratio]
```

Comment:
Looks like we need four more spaces (indentation) here to align with the context.